tests/apis/token.json

```diff
@@ -0,0 +1,155 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "0.0.1",
+        "title": "title"
+    },
+    "securityDefinitions": {
+        "oauth2": {
+            "type": "oauth2",
+            "scopes": {},
+            "flow": "password",
+            "tokenUrl": ""
+        }
+    },
+    "paths": {
+        "/auth": {
+            "post": {
+                "description": "Get token",
+                "operationId": "auth",
+                "parameters": [
+                    {
+                        "name": "grant_type",
+                        "description": "Authorization grant type. Use <code>client_credentials</code> to create a token or <code>refresh_token</code> to refresh a token",
+                        "required": true,
+                        "type": "string",
+                        "enum": ["client_credentials", "refresh_token"],
+                        "in": "query"
+                    },
+                    {
+                        "name": "email",
+                        "description": "The account email. Mandatory if <code>grant_type=client_credentials</code>.",
+                        "required": false,
+                        "type": "string",
+                        "in": "query"
+                    },
+                    {
+                        "name": "password",
+                        "description": "The account password. Mandatory if <code>grant_type=client_credentials</code>.",
+                        "required": false,
+                        "type": "string",
+                        "in": "query"
+                    },
+                    {
+                        "name": "refresh_token",
+                        "description": "The <code>refresh_token</code> obtained in the last successful request to this endpoint.  Mandatory if <code>grant_type=refresh_token</code>.",
+                        "required": false,
+                        "type": "string",
+                        "in": "query"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Token",
+                        "schema": {
+                            "$ref": "#/definitions/Authentication"
+                        }
+                    },
+                    "default": {
+                        "description": "Error",
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                }
+            }
+        },
+        "/v1/_queries": {
```

Comment:
@borisirota This endpoint won't work. Try `GET /project?token=...` and it should be a great test case.